<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HatchetAccountConfig</class>
 <widget class="QWidget" name="HatchetAccountConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources.qrc">:/hatchet-account/hatchet-icon-512x512.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>Connect to your Hatchet account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="loggedInLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="credentialsLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="otpLabel">
       <property name="text">
        <string>Enter One-time
Password (OTP)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="usernameEdit">
       <property name="placeholderText">
        <string>Hatchet username</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Hatchet password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="otpEdit">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
